stories/Icons.js

```diff
@@ -94,81 +73,6 @@ storiesOf('Icons', module)
   .addDecorator(IconsDecorator)
   .addWithJSX('All icons', () => (
     <>
-      <AddPeopleIcon />
-      <AddToBoardIcon />
-      <AllItemsIcon />
-      <ArrowIcon />
-      <BellIcon />
-      <BinocularsIcon />
-      <BoardIcon />
-      <CameraIcon />
-      <CheckIcon />
-      <ChevronLeftIcon />
-      <ClipboardIcon />
-      <CloneCompleteIcon />
-      <CloudUploadIcon />
-      <CloudUploadBoldIcon />
-      <CommentIcon />
-      <CopyIcon />
-      <CreateBoardIcon />
-      <CrossIcon />
-      <DestructIcon />
-      <DetachIcon />
-      <DisableIcon />
-      <DotsIcon />
-      <DownIcon />
-      <DownloadIcon />
-      <DropdownIcon />
-      <DropdownArrowIcon />
-      <EnableIcon />
-      <ErrorIcon />
-      <ExpireTimeIcon />
-      <FolderIcon />
-      <GearIcon />
-      <ImageIcon />
-      <InfoIcon />
-      <KeyIcon />
-      <LogoutIcon />
-      <MailIcon />
-      <MoveIcon />
-      <NewWindowIcon />
-      <NotesIcon />
-      <NotesBoldIcon />
-      <OthersIcon />
-      <PadlockIcon />
-      <PadlockBoldIcon />
-      <PadlockLockIcon />
-      <PadlockLockOutlineIcon />
-      <PadlockUnlockIcon />
-      <PadlockUnlockOutlineIcon />
-      <PenIcon />
-      <PrivateIcon />
-      <ProfileIcon />
-      <PublicIcon />
-      <PublicFolderIcon />
-      <RemoveTagIcon />
-      <SearchIcon />
-      <SearchBoldIcon />
-      <SendIcon />
-      <ShareIcon />
-      <SharedIcon />
-      <StarIcon />
-      <TagFilledIcon />
-      <TagsIcon />
-      <TeamIcon />
-      <TrashBinIcon />
-      <UpIcon />
-      <UploadIcon />
-      <UrlIcon />
-      <VerticalDotsIcon />
-      <ViewGridIcon />
-      <ViewListIcon />
-      <WalletIcon />
-      <WarningIcon />
-      <WatchFolderIcon />
-      <ZoomCloseIcon />
-      <ZoomInIcon />
-      <ZoomOutIcon />
-      <ZoomResetIcon />
+      {iconsArray.map(getIcon)}
```

Comment:
Can we run here a function called `getIcons()` that will run this `iconsArray.map()` for us, and also do the job that `getIcon` is doing right now? ;)